zetaclient/zetabridge/zetacore_bridge.go

```diff
@@ -221,28 +221,28 @@ func (b *ZetaCoreBridge) UpdateZetaCoreContext(coreContext *corecontext.ZetaCore
 
 	// check and update chain params for each chain
 	for _, chainParam := range chainParams {
+		if !chainParam.GetIsSupported() {
+			b.logger.Info().Msgf("Chain %d is not supported yet", chainParam.ChainId)
```

Comment:
Does it mean it is printed every time we enter to ticker to update the core context?
If so we should turn it to debug logs to makes logs less verbose

Reply:
I turned them into a sampled logger so they still get printed in Datadog but less frequently.
https://github.com/zeta-chain/node/pull/1969/commits/8b540783471f1309881e8b274ba4750626bb3842